packages/edit-site/src/components/global-styles/font-library-modal/context.js

```diff
@@ -93,7 +93,7 @@ function FontLibraryProvider( { children } ) {
 	 *
 	 * @param {Array} fonts - The font families that will be saved to the database.
 	 */
-	const saveFontFamilies = async ( fonts ) => {
+	const saveFontFamilies = async ( fonts, shouldAutoSave = false ) => {
```

Comment:
```suggestion
	const saveFontFamilies = async ( fonts, persist = false ) => {
```

Considering that autosave can have a different meaning in WP, let's use another name for the argument. Additionally, let's update the JSDoc block and include a new argument.

Reply:
Thanks for the suggestions. I have made the changes in the latest commit 🚀